server/tinylicious/src/services/levelDb.ts

```diff
@@ -63,3 +63,16 @@ export class LevelDb extends EventEmitter implements IDb {
         }
     }
 }
+
+export class LevelDbFactory implements IDbFactory {
+    private readonly db;
+
+    constructor(path: string) {
+        this.db = new LevelDb(path);
+    }
+
+    public async connect(): Promise<IDb> {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Curious, why disable this eslint rule? Instead, doesn't

`private readonly db: LevelDb;`

solve the issue?